models/published/direct_care/C_LTCS/cltcs_score_frailty.sql

```
@@ -71,31 +105,113 @@ encoding_features as (
+        least(greatest(zeroifnull(scaled_score_asc_indicators), -3), 3) as clipped_score_asc_indicators
+    from composite_scores
+),
+
+reweighted_scores as (
+    select *,
+     ( 1 * clipped_score_clinical_complexity + 1 * clipped_score_clinical_frailty + 1 * clipped_score_medicines_management + 1 * clipped_score_emergency_use + 1 * clipped_score_residential_social_factors + 1 * clipped_score_wider_care_engagement + 1 * clipped_score_asc_indicators) / 7 as raw_score_frailty
+    from clipped_scores
+)
+select *,
+    round((raw_score_frailty + 3) / 6.0 * 100, 1) - (50 * score_exclusions) as score_frailty
+from reweighted_scores
```

SQLFluff (4.2.2)

[error] Blank line expected but not found after CTE closing bracket.

(LT08)